docs/guide/writing-tasks.md

```diff
@@ -591,17 +591,21 @@ CIRRUS_ENV_SENSITIVE | Set to `true` to mask all variable values written to the
+| CIRRUS_ESCAPING_PROCESSES  | not set                                                                                                                                                | Set this variable to prevent the agent from terminating the processes spawned in each non-background instruction after that instruction ends. By default, the agent tries it's best to garbage collect these processes and their standard input/output streams. It's generally better to use a [Background Script Instruction](#background-script-instruction) instead of this variable to achieve the same effect. |
+| CIRRUS_WINDOWS_ERROR_MODE  | not set                                                                                                                                                | Set this value to force all processes spawned by the agent to call the equivalent of [`SetErrorMode()`](https://docs.microsoft.com/en-us/windows/win32/api/errhandlingapi/nf-errhandlingapi-seterrormode) with the provided value (for example, `0x8001`) before beginning their execution.                                                                                                                         |
+| CIRRUS_VAULT_URL           | not set                                                                                                                                                | Address of the Vault server expressed as a URL and port (for example, `https://vault.example.com:8200/`), see [HashiCorp Vault Support](#hashicorp-vault-support).                                                                                                                                                                                                                                                  |
+| CIRRUS_VAULT_NAMESPACE     | not set                                                                                                                                                | A [Vault Enterprise Namespace](https://developer.hashicorp.com/vault/docs/enterprise/namespaces) to use when authenticating and reading secrets from Vault.                                                                                                                                                                                                                                                         |
+| CIRRUS_VAULT_AUTH_PATH     | `jwt` for JWT, etc.                                                                                                                                    | Alternative auth method mount point, in case it was mounted to a non-default path.                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
I think the ` for JWT, etc.` part can be omitted? "etc." confused me a bit. 😅

Reply:
See https://github.com/cirruslabs/cirrus-ci-docs/pull/1088/commits/26fa9746b7853ae8286f9384c23e379d2e1ee2a5.